pkg/csi-common/driver.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package csicommon
 
 import (
-	"github.com/golang/glog"
+	log "github.com/sirupsen/logrus"
+
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 )
 
+// CSIDriver ...
```

Comment:
I would complete this comment, maybe `CSIDriver provides a container storage interface driver implementation for the secrets store project`

Reply:
thanks, done!